src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -150,36 +150,56 @@ public static String getIndividualLastName(Entity entity) {
         return entity.getName().getLastName();
     }
 
-    public static Optional<String> getIndividualPreferredHearingChannel(String hearingType, HearingSubtype hearingSubtype) {
+    public static String getIndividualPreferredHearingChannel(String hearingType,
+                                                                        HearingSubtype hearingSubtype,
+                                                                        HearingOptions hearingOptions) {
         if (hearingType == null || hearingSubtype == null) {
-            return Optional.empty();
+            return null;
```

Comment:
This should probably be an IllegalStateException too

Reply:
Done